--- harfbuzz-1.1.3/src/hb-uniscribe.cc.orig	2016-01-06 13:29:29.337284954 +0100
+++ harfbuzz-1.1.3/src/hb-uniscribe.cc	2016-01-12 17:19:29.371811035 +0100
@@ -42,6 +42,9 @@
 #define HB_DEBUG_UNISCRIBE (HB_DEBUG+0)
 #endif
 
+#ifndef E_NOT_SUFFICIENT_BUFFER
+#define E_NOT_SUFFICIENT_BUFFER ((HRESULT)0x8007007A)
+#endif
 
 static inline uint16_t hb_uint16_swap (const uint16_t v)
 { return (v >> 8) | (v << 8); }
